src/Antiresonant.jl

```diff
@@ -90,4 +92,211 @@ __neff(A, B, C, D, σ, nco, loss::Val{true}) = nco*(1 - A*σ^2 - B*σ^3 - C*σ^4
 __neff(A, B, C, D, σ, nco, loss::Val{false}) = real(nco*(1 - A*σ^2 - B*σ^3 - C*σ^4))
 __neff(A, B, C, D, σ, nco, loss::Number) = nco*(1 - A*σ^2 - B*σ^3 - C*σ^4 + 1im*loss*D*σ^4)
 
-end
\ No newline at end of file
+
+struct VincettiMode{mT<:Capillary.MarcatiliMode, LT} <: AbstractMode
+    m::mT
+    t::Float64 # tube wall thickness
+    r_ext::Float64 # tube radius
+    Ntubes::Int # number of tubes
+    cladn::Float64 # cladding refractive index (constant, defaults to 1.45)
+    Nterms::Int # number of terms (cladding modes) to include in the sum
+    loss::LT # Val{true}(), Val{false}() or a number (scaling factor)
+end
+
+"""
+    VincettiMode(a, args...; wallthickness, tube_radius, Ntubes, cladn, Nterms,
+                             loss=true, kwargs...)
+
+Create a mode with Marcatili-like mode fields but loss, dispersion and effective area given
+by the semi-empirical model developed by Vincetti et al. in refs [1-3]. Arguments are identical
+to `Capillary.MarcatiliMode` but with the following additions/changes as keyword arguments:
+
+# Mandatory keyword arguments
+- `wallthickness` : wall thickness of the resonators (cladding capillaries)
+- `tube_radius` : radius of the resonators
+- `Ntubes` : number of resonators
+
+# Optional keyword arguments
+- `cladn` : refractive index of the resonators. This must be a constant for the model.
+            Defaults to 1.45
+- `Nterms` : number of resonator dielectric modes to include in the model. Defaults to 8.
+- `loss` : can be `true` or `false` to switch loss on/off, or a `Real` to scale the loss.
+
+To specify the gap between resonators, calculate the core radius with [`getRco(r_ext, N, δ)`](@ref).
```

Comment:
I think it would be more useful to suggest using `getr_ext(Rco, N, δ)` to calculate `r_ext`. I expect most people want to keep `Rco` fixed?

Reply:
now suggesting both